src/main/java/net/glowstone/util/SoundUtil.java

```diff
@@ -0,0 +1,40 @@
+package net.glowstone.util;
+
+import net.glowstone.GlowWorld;
+import net.glowstone.entity.GlowPlayer;
+import org.bukkit.Location;
+import org.bukkit.Sound;
+
+import java.util.Arrays;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class SoundUtil {
+
+    private static final ThreadLocalRandom rand = ThreadLocalRandom.current();
```

Comment:
Wouldn't this result in the same ThreadLocalRandom being used across all threads?